### PR TITLE
Spigushe master (rebased)

### PR DESCRIPTION
**Pull request by lionel-panhaleux**
Rewrote commits for ease of review

### CODE REVIEW

**Review by Spigushe — APPROVED** [review]
**Add decklist-converter**
When I commited this part, I was not sure the right place was the codex instead of krcg api. I'm still wondering if it is the right place but I think that putting it here will help the further development of the codex by integrating more lists. The KRCG API should be the go to nevertheless.

**JSON decklists**
This part is fine

**Templates**
Work that might be left to do here is :
- [ ] Re-check every errata
- [ ] Re-check clan symbol (right symbol and need of a symbol for multi-clan crypts)

**Review by Spigushe — COMMENTED** [review]
Work that might be left to do here is :
- [ ] Check every errata
- [ ] Check clan symbol (right symbol and need of a symbol for multi-clan crypts)

**Comment by Spigushe** [comment]
Just to warn you that you removed all erratas that i have put in every deck
I think they should be kept at all costs

**Comment by lionel-panhaleux** [comment]
- I'm OK with the 419 Operation review - I made minor changes on it.
- For now, when a deckles has no clan, it gets no icon on the title, I'm alright with that.
- Normally, erratas should be there (at least they are on my side)

File: codex_of_the_damned/templates/archetypes/bibliodeque/shas-threesome.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+{% import "archetypes/_layout-erratum.html" as erratum with context %}
+{% import "archetypes/bibliodeque/shas-threesome.json" as deck_json %}
+{% import "archetypes/bibliodeque/_layout.html" as layout with context %}
+
+{% block meta %}
+<meta property="decklist" content='{{ deck_json | tojson }}' />
+{% endblock %}
+
+{% block archetype_name %}
+<h1>Sha's Threesome</h1>
```
Comment (Spigushe) [inline codex_of_the_damned/templates/archetypes/bibliodeque/shas-threesome.html:11]: This deck needs a change of name imo

File: codex_of_the_damned/templates/archetypes/bibliodeque/weenie-ani-wall.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{% extends "archetypes/nananimalism.html" %}
+{% import "archetypes/bibliodeque/_layout.html" as layout with context %}
+
+{% block archetype_name %}
+<h1>Weenie ANI <em>Wall</em></h1>
```
Comment (Spigushe) [inline codex_of_the_damned/templates/archetypes/bibliodeque/weenie-ani-wall.html:5]:
```suggestion
<h1>Weenie ANI</h1>
```

File: codex_of_the_damned/navigation.py [hunk 3]
```diff
@@ -138,6 +138,113 @@ def walk(self, path=None, top=None, ante=None, post=None):
...
+                        Nav("Tuna Tunnel"),
+                        Nav("Tunnel Runners"),
+                        Nav("Tupdogs"),
+                        Nav("Turbo Sensory"),
+                        Nav("Ublo's Harem"),
+                        Nav("Una Circus"),
+                        Nav("Unacceptable Boon"),
+                        Nav("Unmada Crescendo"),
+                        Nav("Warding Pan"),
+                        Nav("Weenie ANI Wall"),
```
Comment (Spigushe) [inline codex_of_the_damned/navigation.py:241]:
```suggestion
                        Nav("Weenie ANI"),
```

File: codex_of_the_damned/navigation.py [hunk 4]
```diff
@@ -138,6 +138,113 @@ def walk(self, path=None, top=None, ante=None, post=None):
...
+                        Nav("Ozmo 94"),
+                        Nav("Palla Grande"),
+                        Nav("Piper War Ghoul"),
+                        Nav("Red Army"),
+                        Nav("Reversal"),
+                        Nav("Richter & Friends"),
+                        Nav("Rock Cats"),
+                        Nav("Sarrasine Velvet Tongue"),
+                        Nav("Seraph's Ambitions"),
+                        Nav("Sha's Threesome"),
```
Comment (Spigushe) [inline codex_of_the_damned/navigation.py:219]:
```suggestion
                        Nav("Sha Trio"),
```

File: codex_of_the_damned/templates/archetypes/bibliodeque/shas-threesome.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
+{% extends "archetypes/_layout.html" %}
+{% import "archetypes/_layout-erratum.html" as erratum with context %}
+{% import "archetypes/bibliodeque/shas-threesome.json" as deck_json %}
```
Comment (Spigushe) [inline codex_of_the_damned/templates/archetypes/bibliodeque/shas-threesome.html:3]:
```suggestion
{% import "archetypes/bibliodeque/shas-trio.json" as deck_json %}
```

File: codex_of_the_damned/templates/archetypes/bibliodeque/shas-threesome.html (hunk 1 above)
Comment (Spigushe) [inline codex_of_the_damned/templates/archetypes/bibliodeque/shas-threesome.html:11]:
```suggestion
<h1>Sha Trio</h1>
```

File: codex_of_the_damned/templates/archetypes/bibliodeque/shas-threesome.json [hunk 6]
```diff
@@ -0,0 +1,237 @@
+{
+  "name": "Sha's Threesome",
```
Comment (Spigushe) [inline codex_of_the_damned/templates/archetypes/bibliodeque/shas-threesome.json:2]:
```suggestion
  "name": "Sha Trio",
```

**Pull request merged**